config/owasp/suppressions.xml

```diff
@@ -71,4 +71,7 @@
         <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat-embed-core@.*$</packageUrl>
         <cve>CVE-2023-41080</cve>
     </suppress>
+    <suppress>
+        <cve>CVE-2023-42503</cve>
+    </suppress>
```

Comment:
Please no suppressions when a fix exists for it. We're already using commons-compress v1.23.0 in our build.gradle. Just bump this to 1.24.0